File: 3019-furthest-point-from-origin/furthest-point-from-origin.c
int furthestDistanceFromOrigin(char * moves){
    int lodash = 0;
    int hor = 0;

    for(char *move = moves; *move != '\0'; *move++){
        if(*move == '_'){
            lodash++;
        } else if(*move == 'L'){
            hor--;
        } else {
            hor++;
        }
    }

    return abs(hor) + lodash;
}

File: 3019-furthest-point-from-origin/furthest-point-from-origin.js
/**
 * @param {string} moves
 * @return {number}
 */
var furthestDistanceFromOrigin = function(moves) {
    let hor = 0;
    let lodash = 0;
    for(let i = 0; i < moves.length; i++){
        if(moves[i] == 'L'){
            hor--;
        } else if(moves[i] == 'R'){
            hor++;
        } else {
            lodash++;
        }
    }

    return Math.max(Math.abs(hor - lodash), Math.abs(hor + lodash));
};